### PR TITLE
Update Quo Vagis

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
Updated the checksum and cleaning data which was frankly horribly out of date, same issue as the previous PR idk if a specific bash tag applies to consumable effects.

I also relocated Quo Vagis under the Gameplay - Quests subsection because I felt like it was the most appropriate location for it in the masterlist.

### CODE REVIEW

**Comment by MacSplody** [comment]
You'll have to rebase this pull request onto branch `v0.21`, it has changes from another PR.
Which could cause merge conflict when #437 is merged.

What are you using btw, Github Desktop or gitbash?

image

**Comment by Superfly-Johnson** [comment]
Gitbash, I wouldn't even know nor want to know how to use github desktop, I'll merge Quo Vagis after The Institute is merged in.

File: masterlist.yaml [hunk 1]
```diff
@@ -4322,6 +4322,18 @@ plugins:
         util: '[FO3Edit v4.0.3g](https://www.nexusmods.com/fallout3/mods/637)'
         nav: 16
 
+  - name: 'BelthansQuoVagis.esm'
+    url:
+      - link: 'https://www.nexusmods.com/fallout3/mods/19305/'
+        name: 'Quo Vagis'
+    tag: [ C.Light ]
+    dirty:
+    # 3.52
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

**Review by MacSplody — APPROVED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -4322,6 +4322,17 @@ plugins:
         util: '[FO3Edit v4.0.3g](https://www.nexusmods.com/fallout3/mods/637)'
         nav: 16
 
+  - name: 'BelthansQuoVagis.esm'
+    url:
+      - link: 'https://www.nexusmods.com/fallout3/mods/19305/'
+        name: 'Quo Vagis'
+    tag: [ C.Light ]
```
Comment (MacSplody) [inline masterlist.yaml]: Tag is not required, it doesn't make any changes to Cell lighting.
It also doesn't need any other tags.
```suggestion
```

**Pull request merged**